client/components/reader-full-post/index.jsx

```diff
@@ -21,19 +23,36 @@ import { fetchPost } from 'lib/feed-post-store/actions';
 import ReaderFullPostHeader from './header';
 
 export class FullPostView extends React.Component {
+	constructor( props ) {
+		super( props );
+		this.handleBack = this.handleBack.bind( this );
+	}
+
+	handleBack() {
+		this.props.onClose && this.props.onClose();
+	}
+
 	render() {
 		/*eslint-disable react/no-danger*/
 		return (
-			<Main className="reader-full-post">
-				<StickyPanel>
-					<div className="reader-full-post__back">
-						<Gridicon icon="arrow-left" />
-					{ translate( 'Back' ) }
-					</div>
+			<ReaderMain className="reader-full-post">
+				<StickyPanel className="reader-full-post__back-container">
+					<Button className="reader-full-post__back" borderless compact onClick={ this.handleBack }>
+						<Gridicon icon="arrow-left" /> { translate( 'Back' ) }
+					</Button>
 				</StickyPanel>
-				<ReaderFullPostHeader post={ this.props.post } />
-				<div dangerouslySetInnerHTML={ { __html: this.props.post.content } } />
-			</Main>
+				<div className="reader-full-post__content">
+					<div className="reader-full-post__sidebar">
```

Comment:
Can we drop this for the moment until author-compact-profile has landed? (#7127)